### PR TITLE
Update Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj config attribute

### PR DESCRIPTION
**Pull request by yophilav**
Update Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj config attribute to fix a build failure due to deprecated configuration fields.

For more info regarding the no-long-support csproj fields: 
https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file
https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5048

### CODE REVIEW

**Review by dylanbronson — APPROVED** [review]
lgtm

**Review by richma-ms — APPROVED** [review]

**Review by ancaantochi — APPROVED** [review]

File: edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj [hunk 1]
```diff
@@ -14,14 +14,19 @@
...
-    <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</PackageIconUrl>
+    <PackageIcon>nugetIcon.png</PackageIcon>
     <PackageProjectUrl>https://github.com/Azure/iotedge</PackageProjectUrl>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageTags>IoT Microsoft Azure IoTEdge Functions</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
+      <None Include="..\..\..\..\..\LICENSE" Pack="true" PackagePath=""/>
+      <!-- Orinal Icon: http://go.microsoft.com/fwlink/?LinkID=288890 -->
```
Comment (philipktlin) [inline edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj:27]: typo
```suggestion
      <!-- Original Icon: http://go.microsoft.com/fwlink/?LinkID=288890 -->
```

File: edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj [hunk 2]
```diff
@@ -14,14 +14,19 @@
...
+      <None Include="..\..\..\..\..\doc\images\nugetIcon.png" Pack="true" PackagePath=""/>
```
Comment (philipktlin) [inline edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj:28]: I didn't find nugetIcon.png based on given path.

**Comment by philipktlin** [comment]
You need to apply the same fix against master branch and release\1.0.10 branches.

File: edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj (hunk 2 above)
Comment (yophilav) [reply edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj:28]: I just added them in this PR.

**Pull request merged**